### PR TITLE
add rule for BusinessInformant.de

### PR DESCRIPTION
**Pull request by numismatika**

### CODE REVIEW

File: src/chrome/content/rules/BusinessInformant.de.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!--
+	Cert mismatch:
+		-  mail
+
+	empty reply:
+		- evm
```
Comment (J0WI) [inline src/chrome/content/rules/BusinessInformant.de.xml]: Works on evm.business-informant.de/event.php

**Comment by J0WI** [comment]
ping @numismatika

File: src/chrome/content/rules/BusinessInformant.de.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+-->
+<ruleset name="BusinessInformant.de">
+
+	<target host="www.business-informant.de" />
+	<target host="evm.business-informant.de" />
+	<target host="links.business-informant.de" />
+	<target host="newsabo.business-informant.de" />
+
+	<test url="http://evm.business-informant.de/event.php" />
```
Comment (J0WI) [inline src/chrome/content/rules/BusinessInformant.de.xml]: You should also add an exclusion for `$` to make Travis happy.

**Comment by numismatika** [comment]
@J0WI travis is happy now

**Pull request merged**